core/client/src/main/java/alluxio/client/block/RetryHandlingBlockWorkerClient.java

```diff
@@ -124,11 +129,11 @@ private void init() throws IOException {
             @Override
             public void run() {
               try {
-                sessionHeartbeat();
+                sessionHeartbeat(new CountingRetry(0));
               } catch (InterruptedException e) {
                 // Do nothing.
               } catch (Exception e) {
-                LOG.error("Failed to heartbeat for session {}", mSessionId, e);
+                LOG.warn("Failed to heartbeat for session {}", mSessionId, e);
```

Comment:
actually, the current approach was double logging (both inside and outside of `sessionHeartbeat`), I only kept the logging inside of it